plugins/helm/yaml-cpp-config.cmake

```diff
@@ -0,0 +1,16 @@
+# - Config file for the yaml-cpp package
+# It defines the following variables
+#  YAML_CPP_INCLUDE_DIR - include directory
+#  YAML_CPP_LIBRARIES    - libraries to link against
+
+@PACKAGE_INIT@
+
+set_and_check(YAML_CPP_INCLUDE_DIR "@PACKAGE_CMAKE_INSTALL_INCLUDEDIR@")
+
+# Our library dependencies (contains definitions for IMPORTED targets)
+include(@PACKAGE_CONFIG_EXPORT_DIR@/yaml-cpp-targets.cmake)
+
+# These are IMPORTED targets created by yaml-cpp-targets.cmake
+set(YAML_CPP_LIBRARIES "@EXPORT_TARGETS@")
+
+check_required_components(@EXPORT_TARGETS@)
```

Comment:
What is the purpose of this file? I don't see it being used at all.

Reply:
`find_package(yaml-cpp)` uses this file to set the `YAML_CPP_INCLUDE_DIR` and `YAML_CPP_LIBRARIES` variables which are used in the `CMakeLists.txt` file of the parser component.

Reply:
`yaml-cpp-config.cmake` is installed with the `libyaml` library, for me is available on the path `/usr/lib/x86_64-linux-gnu/cmake/yaml-cpp/yaml-cpp-config.cmake`.

That is the used version. This local version is not referenced at all and is not used as I have seen.